app/models/user.rb

```diff
@@ -5,9 +5,10 @@ class User < ActiveRecord::Base
          :recoverable, :rememberable, :trackable, :validatable
 
   has_many :lists
+  has_many :items
```

Comment:
Remove `has_many :items` from `User`. Users are related to Items through the Lists.